### DISCUSSION TITLE
Subscribe as both Seller and Buyer using the same subscription id

### DISCUSSION THREAD

**Discussion by V2-Rapido** [ONDC-Official/developer-docs#52] [Q&A]
Hello, 

We are already subscribed as a seller, now we are trying to subscribe as a buyer using the same subscription id by hitting "https://staging.registry.ondc.org/subscribe" using the payload specific to "ops_no: 1".

But we are getting the error below:
{'message': {'ack': {'status': 'NACK'}}, 'error': {'type': 'POLICY-ERROR', 'code': '125', 'path': None, 'message': 'Subscriber id already exists'}}

Any help is appreciated.

**Comment by pratikmazumdar** [comment]
To subscribe as a seller and buyer both, you needed to use ops_no:3 when subscribing, drop a mail to techsupport@ondc.org we will change it for you.

**Reply by olttwa** [reply]
**Correction**: A NP must subscribe using `ops_no:4` as specified in the [official onboarding doc](https://github.com/ONDC-Official/developer-docs/blob/main/registry/Onboarding%20of%20Participants.md#supported-registrations).
